.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": [["@babel/plugin-proposal-nullish-coalescing-operator"]]
```

Comment:
While working on the Cow Explorer integration, we faced the issue of the Webpack compiler yelling at the `index.module.js` file due to a parse error in:

```js
ERROR in ../cow-sdk/dist/index.module.js 734:26
Module parse failed: Unexpected token (734:26)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|       return {
|         result: !!t(e),
>         errors: r.errors ?? void 0
|       };
|     });
 @ ./src/const.ts 15:0-46 212:25-31
 @ ./src/components/layout/LegacyTradeLayout/index.tsx
 @ ./src/components/layout/index.ts
 @ ./src/apps/explorer/ExplorerApp.tsx
 @ ./src/apps/explorer/index.tsx
````

Searching for the root cause, we found out that the version of the `acorn` parser that uses Webpack 4 doesn't understand the nullish coalescing operator that the TS compiler is emitting. So this fix guarantees that the Explorer integration works fine.